feflow/settings/nonequilibrium_cycling.py

```diff
@@ -7,7 +7,11 @@
 from typing import Optional
 from gufe.settings import Settings
 from openff.units import unit
-from pydantic import root_validator
+# Support for both pydantic 1 and 2
+try:
+    from pydantic.v1 import root_validator
+except ImportError:
+    from pydantic import root_validator
```

Comment:
I think this is what we're doing too everywhere.  If we wanted to migrate to v2 only, we'd have to chase upstream to check if anything is v1 only.  I haven't had time to check, but I'd guess that OpenFF + QCStack might be a place where there's significant usage of pydantic (and I have no idea if they've prioritised this).